probe-dictionary/explore.js

```diff
@@ -939,6 +942,12 @@ function getMeasurementCountsPerVersion() {
     };
   }
 
+function releaseProbesCount(data, k) {
```

Comment:
Nice choice to not repeat the logic and put it in a function instead!

I see that this also counts non-release/non-"optout" probes.
Can we name this more descriptively? E.g. `countProbe()`?

Also a small style issue: This is a function that is defined locally, so it should be indented to the surrounding code.